.github/workflows/main.yml

```diff
@@ -0,0 +1,28 @@
+name: CI
```

Comment:
your merging is not proper, because this file already exists in `origin/develop` branch

Reply:
What I meant is that these two commits should be removed from this PR by rebasing,
because these two commits are already merged into master branch.
[`faa141c` (#221)](https://github.com/ros-drivers/usb_cam/pull/221/commits/faa141cdcd5c4969de6f3677521c538645f24bae)
[`b19346b` (#221)](https://github.com/ros-drivers/usb_cam/pull/221/commits/b19346b611eebbe91308e32a9dbc0739dc98369d)

run `git rebase origin/develop -i` and remove the two commits and one new commit [`19ca643` (#221)](https://github.com/ros-drivers/usb_cam/pull/221/commits/19ca643ebef7924b776d13126c19873741cd21a3)

Reply:
Doing, thanks!